orttraining/orttraining/python/orttraining_pybind_state.cc

```diff
@@ -520,14 +520,23 @@ py::class_<TrainingAgent>(m, "TrainingAgent", R"pbdoc(This is the main class use
   py::class_<OrtModuleGraphBuilderConfiguration> module_graph_builder_config(
       m, "OrtModuleGraphBuilderConfiguration",
       R"pbdoc(Configuration information for module graph builder.)pbdoc");
+
+  py::enum_<Severity>(m, "Severity", py::arithmetic(), py::module_local())
```

Comment:
one improvement point is to check if there are existing APIs in this file which could leverage this new enum

Reply:
In this file I don't see where I can use the new enu.

Reply:
OrtRunOptions also is something that can change data from from int to OrtLoggingLevel.